src/utils.ts

```diff
@@ -1,4 +1,5 @@
-import isEqual from 'lodash.isequal'
+import { isDeepStrictEqual } from 'node:util'
```

Comment:
We are working towards making WebdriverIO accessible on more platforms other than just Node.js, hence I would suggest we find a deep equal package and not depend on Node.js primitives.